source/core/owt_base/RawTransport.cpp

```diff
@@ -13,6 +13,19 @@ using boost::asio::ip::tcp;
 
 DEFINE_TEMPLATE_LOGGER(template<Protocol prot>, RawTransport<prot>, "owt.RawTransport");
 
+static constexpr const char kServerCrt[] = "cert/server.crt";
+static constexpr const char kServerKey[] = "cert/server.key";
+static constexpr const char kDHParams[] = "cert/dh2048.pem";
```

Comment:
Provide provide documentation on how to prepare those cert materials.

Reply:
Document added